appengine-standard-java8/springboot-appengine-standard/src/main/resources/application.properties

```diff
@@ -1,14 +1,14 @@
-# Copyright 2018 Google LLC
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-# http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#
+    # Copyright 2017 Google Inc.
```

Comment:
Is there a reason for the initial indent?

Reply:
Yes, a stupid one. The checkstyle rules for licenses have a "\W+" instead of a "\W*" for licenses, so it expects at least one whitespace character before the license lines.

I could fix this in java-repo-tools, but I assume it's there for a reason, and that would require a repo tools release cycle in order to fix these tests.

Reply:
It's probably just an oversight on my part, I originally had it checking only java files but we expanded it latter.